lint errors after clean node_modules install

```
➜  fxa-content-server git:(lint) grunt lint
Running "eslint:app" (eslint) task

/Users/stomlinson/development/fxa-content-server/app/scripts/lib/require-on-demand.js
  69:11  error  Expected indentation of 8 space characters but found 10  indent
  70:11  error  Expected indentation of 8 space characters but found 10  indent
  71:11  error  Expected indentation of 8 space characters but found 10  indent
  73:11  error  Expected indentation of 8 space characters but found 10  indent
  74:9   error  Expected indentation of 6 space characters but found 8   indent

```
```
➜  fxa-content-server git:(require-on-demand-lint) ✗ grunt lint
Running "eslint:app" (eslint) task


Running "eslint:grunt" (eslint) task

/Users/stomlinson/development/fxa-content-server/grunttasks/l10n-generate-pages.js
  49:7  error  Expected indentation of 4 space characters but found 6  indent
  51:7  error  Expected indentation of 4 space characters but found 6  indent
  54:7  error  Expected indentation of 4 space characters but found 6  indent
  55:7  error  Expected indentation of 4 space characters but found 6  indent
  56:7  error  Expected indentation of 4 space characters but found 6  indent
  58:7  error  Expected indentation of 4 space characters but found 6  indent
  59:7  error  Expected indentation of 4 space characters but found 6  indent
  62:7  error  Expected indentation of 4 space characters but found 6  indent
  65:7  error  Expected indentation of 4 space characters but found 6  indent
  85:5  error  Expected indentation of 2 space characters but found 4  indent

✖ 10 problems (10 errors, 0 warnings)
```

--- a/app/scripts/lib/require-on-demand.js
+++ b/app/scripts/lib/require-on-demand.js
@@ -62,8 +62,8 @@ define([
       // be loaded on demand, and the module returned when the promise
       // resolves.
       var getNow = window.require;
-      getNow(['nocache!' + resourceToGet],
-        deferred.resolve.bind(deferred), function (requireErr) {
+      getNow(['nocache!' + resourceToGet], deferred.resolve.bind(deferred),
+        function (requireErr) {
           // RequireJS errors described in
           // http://requirejs.org/docs/api.html#errors
           var errorType = requireErr.requireType || 'UNEXPECTED_ERROR';

--- a/grunttasks/l10n-generate-pages.js
+++ b/grunttasks/l10n-generate-pages.js
@@ -43,8 +43,8 @@ module.exports = function (grunt) {
   grunt.registerTask('l10n-generate-pages', ['l10n-create-json', 'l10n-generate-tos-pp', 'l10n-compile-templates']);
 
 
-  grunt.registerTask('l10n-compile-templates',
-    'Generate localized versions of the static pages', function () {
+  grunt.registerTask('l10n-compile-templates', 'Generate localized versions of the static pages',
+    function () {
 
       var done = this.async();
 

